Sources/BraintreeSEPADebit/BTSEPADebitNonce.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+/// A  payment method nonce representing a SEPA Debit payment.
+@objcMembers public class BTSEPADebitNonce {
+    
+    /// The payment method nonce.
+    public let nonce: String?
+    
+    /// The IBAN last four characters.
+    public let ibanLastFour: String?
+    
+    /// The customer ID.
+    public let customerID: String?
+    
+    /// The `BTSEPADebitMandateType`.
+    public let mandateType: BTSEPADebitMandateType?
+        
+    init(json: BTJSON) {
```

Comment:
I realized this should probably be an optional init right? `init?` I looked at our other modules and the `BTPaymentMethodNonce` objective-c init and it seems like it is nullable.

Reply:
🤔 I think we can to align with the Objective-C inits today, but am going to add a next major version note to remove non-optional internal inits in the future. I am not tied to one way or the other and I think we'd need to make this optional anyways if we can eventually inherit from `BTPaymentMethodNonce` because the super init will be optional which will force us to make this optional as well.

Reply:
I think the `BTPaymentMethodNonce` relies on the `nonce` value not being nil, otherwise it returns `nil` from it's nullable init. We can probably follow the same pattern, make the `nonce` non-optional on `BTSepaDebitNonce` and unwrap it in this `init?`